src/Swagger/Serializer/DocumentationNormalizer.php

```diff
@@ -171,13 +175,22 @@ public function __construct(ResourceMetadataFactoryInterface $resourceMetadataFa
 
         $this->defaultContext = array_merge($this->defaultContext, $defaultContext);
         $this->identifiersExtractor = $identifiersExtractor;
+
+        $this->openApiBackwardCompatibility = $openApiBackwardCompatibility;
+        $this->openApiNormalizer = $openApiNormalizer;
     }
 
     /**
      * {@inheritdoc}
      */
     public function normalize($object, $format = null, array $context = [])
     {
+        if ($object instanceof OpenApi) {
+            // uncomment this for 2.7
+            // @trigger_error('Using the swagger DocumentationNormalizer is deprecated in favor of decorating the OpenApiFactory', \E_USER_DEPRECATED);
```

Comment:
You can maybe uncomment this now. I know that we must not introduce deprecations in patch version, but this is more a bug fix than something else, it was missing in 2.6.0 and it currently doesn't work at all, so I can we can add it here.